.gitignore

```diff
@@ -58,3 +58,8 @@ addon.json
 .luarc.json
 
 ttt2.jpg
+
+# redundant source files
+*.dx80.vtx
+*.sw.vtx
+*.xbox.vtx
```

Comment:
```suggestion

# redundant source files
*.dx80.vtx
*.sw.vtx
*.xbox.vtx
```

Please don't do this.

These files aren't expected to exist locally so they should not be ignored by git.

Reply:
the files are generated by most model compilers, if someone else were to recompile or compile new files for this addon then the redundant files could potentially be reintroduced

Reply:
That is for reviewers to catch. 

With this it is way easier for the local directory content to diverge from the upstream repository content. Possibly leading to changes in behaviour.